internal/driver/onvifdiscovery.go

```diff
@@ -133,10 +133,10 @@ func (d *Driver) createDiscoveredDevice(onvifDevice onvif.Device) (sdkModel.Disc
 		device.Protocols[OnvifProtocol][LastSeen] = time.Now().Format(time.UnixDate)
 		device.Protocols[OnvifProtocol][FriendlyName] = devInfo.Manufacturer + " " + devInfo.Model
 
-		// Spaces are not allowed in the device name
+		//Spaces and slashes are not allowed in the device name
```

Comment:
typically we have a space after the `//` at the beginning of our comments. I see this problem in a couple places.